sandbox/modules/identity/src/main/java/org/opensearch/identity/configuration/ConfigurationRepository.java

```diff
@@ -102,20 +111,44 @@ public void run() {
                                     createSecurityIndexIfAbsent();
                                     waitForSecurityIndexToBeAtLeastYellow();
 
-                                    ConfigHelper.uploadFile(
-                                        client,
-                                        cd + "internal_users.yml",
-                                        identityIndex,
-                                        CType.INTERNALUSERS,
-                                        DEFAULT_CONFIG_VERSION
-                                    );
+                                    String adminPassword = ConfigHelper.generateCommonLangPassword();
+                                    LOGGER.always().log("Generated a random admin password: {}", adminPassword);
```

Comment:
Probably don't ever want a password in a log.

Reply:
Hi @dblock, thanks for the review! I understand the concern right here, and I will sync up with the team again to see if we can have a work around.